config/horizon.php

```diff
@@ -54,6 +54,19 @@
 
     'prefix' => env('HORIZON_PREFIX', 'horizon:'),
 
+    /*
+    |--------------------------------------------------------------------------
+    | Horizon Site Link
+    |--------------------------------------------------------------------------
+    |
+    | The URL set here will enable a new point in the sidebar of the Horizon
+    | dashboard which allows you to go back to your site, without having to
+    | modify the URL in the browsers' address bar or clicking back a few times.
+    |
+    */
+
+    'site_link' => url('/'),
```

Comment:
Should we use `env()`? horizon domain and the app domain can't be the same all the time?